testsuite/examples/test_framework_GRASS_GIS_with_NC.conf

```diff
@@ -30,12 +30,12 @@ SERVERDIR="/var/www/html/grassgistestreports"
 
 # read module tests list from file
 # may be no|yes
-READ_MODULE_TEST_LIST=yes
+READ_MODULE_TEST_LIST=no
 # file with list of tests to be run
-TEST_LIST_IN=/src/grass_build/testsuite/examples/test.list
+TEST_LIST_IN="${GRASSSRC}/path/to/test.list"
```

Comment:
No it doesn't exist. And if someone sets `READ_MODULE_TEST_LIST `to `yes` it will not work. But would't the same apply to line 20 where a configure script is called? As this file is a configuration in an example folder it might not be too bad but I changed it.